content/src/certifications/esbe.md

```diff
@@ -27,10 +28,12 @@ isSbe: false
 
 ## Application Needs
 
-* Preferences are given to businesses whose principal place of business is in New Jersey but will accept applications from out-of-state businesses
-* Requires submission of documents to support personal net worth (PNW) and economic disadvantaged status.
-* Applicant(s) must participate in the on-site interview proceedings.
+- Preferences are given to businesses whose principal place of business is in New Jersey but will accept applications from out-of-state businesses
+- Requires submission of documents to support personal net worth (PNW) and economic disadvantaged status.
+- Applicant(s) must participate in the on-site interview proceedings.
+
+:::callout{ showHeader="true" headerText="" showIcon="true" calloutType="conditional" }
```

Comment:
pebble - in the python script, we are setting showIcon to false, what is causing it to be true here? This is occurring in multiple places.

Reply:
Manual changes. Any place that gets the default header I've chosen for it to get an icon by default.